src/components/swap/TradePrice.tsx

```diff
@@ -26,7 +26,7 @@ export default function TradePrice({ price, showInverted, setShowInverted }: Tra
     <Text
       fontWeight={500}
       fontSize={14}
-      color={theme.text2}
+      // color={theme.text2}
```

Comment:
Disabling the color prop, so we can control it exclusively with CSS.

Reply:
also this needs to be modded..

Reply:
Reverted changes on this file.